DiffusionPreprocessing/DiffPreprocPipeline_PreEddy.sh

```diff
@@ -432,36 +456,41 @@ main()
 	log_Msg "Create two files for each phase encoding direction"
 
 	Paired_flag=0
-	for (( j=1; j<${Pos_count}; j++ )) ; do
-		CorrVols=`min ${NegVols[${j}]} ${PosVols[${j}]}`
-		${runcmd} echo ${CorrVols} ${PosVols[${j}]} >> ${outdir}/eddy/Pos_SeriesVolNum.txt
-		if [ ${PosVols[${j}]} -ne 0 ]
-		then
-			${runcmd} echo ${CorrVols} >> ${outdir}/rawdata/${basePos}_SeriesCorrespVolNum.txt
-			if [ ${CorrVols} -ne 0 ]
-			then
+	for ((j = 1; j < ${Pos_count}; j++)); do
+		CorrVols=$(min ${NegVols[${j}]} ${PosVols[${j}]})
+		${runcmd} echo ${CorrVols} ${PosVols[${j}]} >>${outdir}/eddy/Pos_SeriesVolNum.txt
+		if [ ${PosVols[${j}]} -ne 0 ]; then
+			${runcmd} echo ${CorrVols} >>${outdir}/rawdata/${basePos}_SeriesCorrespVolNum.txt
+			if [ ${CorrVols} -ne 0 ]; then
 				Paired_flag=1
 			fi
 		fi
 	done
 
-	for (( j=1; j<${Neg_count}; j++ )) ; do
-		CorrVols=`min ${NegVols[${j}]} ${PosVols[${j}]}`
-		${runcmd} echo ${CorrVols} ${NegVols[${j}]} >> ${outdir}/eddy/Neg_SeriesVolNum.txt
-		if [ ${NegVols[${j}]} -ne 0 ]
-		then
-			${runcmd} echo ${CorrVols} >> ${outdir}/rawdata/${baseNeg}_SeriesCorrespVolNum.txt
+	for ((j = 1; j < ${Neg_count}; j++)); do
+		CorrVols=$(min ${NegVols[${j}]} ${PosVols[${j}]})
+		${runcmd} echo ${CorrVols} ${NegVols[${j}]} >>${outdir}/eddy/Neg_SeriesVolNum.txt
+		if [ ${NegVols[${j}]} -ne 0 ]; then
+			${runcmd} echo ${CorrVols} >>${outdir}/rawdata/${baseNeg}_SeriesCorrespVolNum.txt
 		fi
 	done
 
-	if [ ${Paired_flag} -eq 0 ] ; then
+	if [ ${Paired_flag} -eq 0 ]; then
 		log_Msg "Wrong Input! No pairs of phase encoding directions have been found!"
 		log_Msg "At least one pair is needed!"
 		exit 1
```

Comment:
Preceding two lines should use `log_Err` and `log_Err_Abort`, respectively.  Then delete the `exit 1` line.